File: 251003/Habrus/publisher/src/main/java/by/bsuir/distcomp/dto/response/ArticleResponseTo.java
package by.bsuir.distcomp.dto.response;

import com.fasterxml.jackson.annotation.JsonRootName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonRootName("article")
public class ArticleResponseTo {

    private Long id;
    private Long authorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;

}

File: 251002/Nikonorova/lab1_rest/DTOs/ArticleResponseTo.cs
using System.Text.Json.Serialization;

namespace distcomp.DTOs
{
    public class ArticleResponseTo
    {
        [JsonPropertyName("id")]
        public long Id { get; set; }
        [JsonPropertyName("creatorId")]
        public long CreatorId { get; set; }
        [JsonPropertyName("title")]
        public string Title { get; set; }
        [JsonPropertyName("content")]
        public string Content { get; set; }
        [JsonPropertyName("created")]
        public DateTime Created { get; set; }
        [JsonPropertyName("modified")]
        public DateTime Modified { get; set; } 
    }
}
